Leibit.Core/Data/sued/TF.xml

```diff
@@ -0,0 +1,315 @@
+      <alternative track="6"/>
+      <alternative track="1"/>
+    </track>
+    <track name="53">
+      <block name="77G53"/>
+      <alternative track="54"/>
+      <alternative track="55"/>
+      <alternative track="56"/>
+      <alternative track="57"/>
+    </track>
+    <track name="54">
+      <block name="77G54"/>
+      <alternative track="53"/>
+      <alternative track="55"/>
+      <alternative track="56"/>
+      <alternative track="57"/>
+    </track>
+    <track name="55">
+      <block name="77G55"/>
+      <alternative track="54"/>
+      <alternative track="53"/>
+      <alternative track="56"/>
+      <alternative track="57"/>
+    </track>
+    <track name="56">
+      <block name="77G56"/>
+      <alternative track="54"/>
+      <alternative track="55"/>
+      <alternative track="53"/>
+      <alternative track="57"/>
+    </track>
+    <track name="57">
+      <block name="77G57"/>
+      <alternative track="54"/>
+      <alternative track="55"/>
+      <alternative track="56"/>
+      <alternative track="53"/>
+    </track>
+  </station>
+
+  <station name="Friedrichshafen Hafen" short="TFH" refNr="77" scheduleFile="fn-haf__.abf" localOrderFile="fn-haf__.abf">
+    <track name="152">
+      <block name="7G152"/>
+    </track>
+    <track name="153">
+      <block name="7G153"/>
+    </track>
+  </station>
+  
+  <station name="Friedrichshafen-Ost" short="TFOS" refNr="77" scheduleFile="fn-ost__.abf">
+    <track name="P2" calculateDelay="false">
+      <block name="77BP2"/>
+    </track>
+  </station>
+
+  <station name="Meckenbeuren" short="TMK" refNr="78" scheduleFile="g78_____.abf" localOrderFile="bf78____.abf">
+    <track name="2">
+      <block name="78G2"/>
+      <alternative track="3"/>
+    </track>
+    <track name="3">
+      <block name="78G3"/>
+      <alternative track="2"/>
+    </track>
+  </station>
+
+  <station name="Ravensburg" short="TRB" refNr="79" scheduleFile="g79_____.abf" localOrderFile="bf79____.abf">
+    <track name="1">
+      <block name="79G1"/>
+      <alternative track="2"/>
+      <alternative track="3"/>
+      <alternative track="4"/>
+    </track>
+    <track name="2">
+      <block name="79G2"/>
+    </track>
+    <track name="3">
+      <block name="79G3"/>
+    </track>
+    <track name="4" isPlatform="false">
+      <block name="79G4"/>
+    </track>
+    <track name="8" isPlatform="false">
+      <block name="79G8"/>
+      <alternative track="2"/>
+      <alternative track="3"/>
+      <alternative track="4"/>
+    </track>
+  </station>
+
+  <station name="Weingarten-Berg" short="TWEB" refNr="40" scheduleFile="weinga__.abf">
+    <track name="N1" calculateDelay="false">
+      <block name="40BN1"/>
+    </track>
+    <track name="P" calculateDelay="false">
+      <block name="79BP"/>
+    </track>
+  </station>
+  
+  <station name="Niederbiegen" short="TNB" refNr="40" scheduleFile="g40_____.abf" localOrderFile="bf40____.abf">
+    <track name="1" isPlatform="false">
```

Comment:
Es stimmt zwar, dass diese Gleise keinen Bahnsteig haben. Die Einstellung ist aber eher für Üst's und Abzweige gedacht. Zugegebenermaßen etwas unglücklich benannt...

Bitte das noch entfernen, auch bei Gl. 4+7.

Reply:
Das Gleiche gilt vermutlich auch für Ravensburg Gl. 4 + 8?

Reply:
Ja, genau. Hab ich wohl auch übersehen 😕 In Brandbühl ist es korrekt gesetzt.